src/components/MovieDescription/MovieDescription.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { render } from '@testing-library/react';
+import { testMovie } from '../../utils/testData';
+import { IMovie } from '../../utils/interfaces';
+import { MovieDescription } from './MovieDescription';
+
+test('should render an image', () => {
```

Comment:
No need to check each property being rendered individually, you can go for multiple `expect`s in a single test instead.